contracts/contracts/ZkSync.sol

```diff
@@ -206,9 +206,9 @@ contract ZkSync is UpgradeableMaster, Storage, Config, Events, ReentrancyGuard {
         delegateAdditional();
     }
 
-    /// @notice Deposit ETH to Layer 2 - transfer ether from user into contract, validate it, register deposit
+    /// @notice Deposit RBTC to Layer 2 - transfer ether from user into contract, validate it, register deposit
     /// @param _zkSyncAddress The receiver Layer 2 address
-    function depositETH(address _zkSyncAddress) external payable {
+    function depositRBTC(address _zkSyncAddress) external payable {
```

Comment:
@rsksmart/rif-rollup-tools surely it's a breaking change for the SDK

Reply:
no longer as I understand. @OnyekaIjeh ?